### DISCUSSION TITLE
What's the difference of derive from Iden and DeriveIden?

### DISCUSSION THREAD

**Discussion by npuichigo** [SeaQL/sea-query#732] [Q&A]
```
#[derive(Iden)]
enum Post {
    Table,
    Id,
    Title,
    Text,
}
```
vs 
```
#[derive(DeriveIden)]
enum Post {
    Table,
    Id,
    Title,
    Text,
}
```
and which is perfered?

**Comment by tyt2y3** [comment]
They both implement the same `Iden` trait.

`DeriveIden` is exported by SeaORM, which makes it possible to do things without importing SeaQuery. So if you are using SeaQuery directly, use `Iden` or `IdenStatic`. SeaQuery's `Iden` macro also has more macro attributes, e.g. `#[iden(flatten)]`.
